worklist/views.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+from django.shortcuts import render
+from worklist.models import WorkList
+
+
+# View to create a worklist
+def create_worklist(request):
+    content = {}
+    if request.method == 'POST':
+        content = {'error': 0, 'message': 'Successfully saved worklist'}
+        data = {'name': request.POST.get('name', ''),
+                'theme': request.POST.get('theme', ''),
+                'description': request.POST.get('description', ''),
+                'created_by': request.POST.get('created_by', ''),
+                'psid': request.POST.get('psid', ''),
```

Comment:
Also here, do we really want to allow a worklist to be created with `''` for psid?

Reply:
Corrected it.